e2e/app/text-labels.ts

```diff
@@ -14,15 +14,27 @@ export enum WorkspaceAccessLevel {
 
 // Options in dropdown select.
 export enum Option {
```

Comment:
alphabetized

Reply:
I would call this something like MenuOption so that it's not confusing when used in other files. `Option` looks like it could be, like, xpath options or some other commonly used option

Reply:
👍